application/templates/pages/about/performance.md

```diff
@@ -0,0 +1,35 @@
+Performance indicators showing how the planning data platform is making it easier to find, use and trust planning and housing data in England.
+
+Last updated 19 July 2024.
+
+## Availability
+
+### 89 datasets on the platform
+
+**00 national datasets** available from **00 organisations**. 0 added last month (+/-0%).
+
+**00 datasets** provided by **00 local planning authorities**. 0 provided last month (+/-0%).
```

Comment:
It could be confusing if we’re calling conservation-area data from Lambeth a dataset here, but elsewhere dataset refers to all conservation areas. 

Could we make it clearer by using endpoint as well?

So the topline availability stat could be, for example “_364 endpoints for 34 datasets on the platform_”, and the more detailed one “_321 endpoints for 8 datasets provided by 73 LPAs_”

This could apply to all the dataset references in the rest of the metrics

Reply:
Good point. I've changed it to `data sources` and added how many datasets were provided by LPAs.